<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ReplaceMe</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>REPLACE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="getNameBTN">
       <property name="maximumSize">
        <size>
         <width>16</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="replaceTXT">
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>WITH</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="withTXT"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>PREFIX</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="prefixTXT"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>SUFFIX</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="suffixTXT"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QRadioButton" name="selectedRAD">
       <property name="text">
        <string>SELECTED NODES</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="allRAD">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>ALL NODES</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="childrenRAD">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>CHILDREN</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="numCHK">
       <property name="text">
        <string>NUMBER TAIL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="numEDT">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>01</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>DELETE FIRST</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="deleteFirstSPN">
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>DELETE LAST</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="deleteLastSPN">
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="renameBTN">
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>RENAME</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="testBTN">
       <property name="maximumSize">
        <size>
         <width>19</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Print test</string>
       </property>
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>numCHK</sender>
   <signal>clicked(bool)</signal>
   <receiver>numEDT</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
